pkg/stanza/operator/helper/emitter.go

```diff
@@ -99,6 +99,31 @@ func (e *LogEmitter) Stop() error {
 	return nil
 }
 
+// ProcessBatch emits the entries to the consumerFunc
+func (e *LogEmitter) ProcessBatch(ctx context.Context, entries []*entry.Entry) error {
+	if oldBatch := e.appendEntries(entries); len(oldBatch) > 0 {
+		e.consumerFunc(ctx, oldBatch)
+	}
+
+	return nil
+}
```

Comment:
If we do batching upstream, I would skip more batching here:
1. This cannot be controlled.
2. User can use batchprocessor/exporterhelper etc.